src/ast.rs

```diff
@@ -405,6 +445,55 @@ impl Tag {
             _ => (),
         }
     }
+
+    fn sanitize_string(string: &String) -> Option<String> {
+        let re = Regex::new(r"^[a-zA-Z_][a-zA-Z0-9_]*").unwrap();
+        let mut sanitized = string.replace(".", "_").replace("-", "_");
+        if sanitized.chars().next().unwrap().is_numeric() {
+            sanitized = format!("_{}", sanitized);
+        };
+        if re.is_match(&sanitized) {
+            Some(sanitized)
+        } else {
+            None
+        }
+    }
+
+    // If the current tag is an object, remove keyword properties and apply
```

Comment:
If you use `///` (three slashes) it becomes a doc comment and gets rendered into the final docs.